docs/guide.md

```diff
@@ -439,3 +439,13 @@ if (!storedToken) {
   localStorage.setItem('infos', JSON.stringify(infos))
 }
 ```
+
+### Reset a client
+
+When a user logout, we would like remove all reference on this instance and
+remove all user data. You can just use `cozyClient.unregister()`.
+
+This function make 3 actions:
+- Delete OAuth Client on stack
+- Remove pouch databases
+- Remove all reference on Oauth client
```

Comment:
s/all reference/all references

I would keep this bit for a more technical doc. In the guide, I think we should only keep relevant information for the user. Each link can do its own `reset` so technically you could have a `reset` that would not delete PouchDB databases.

The information that it removes pouch databases could be put on the docstring of PouchLink's reset.